frontend/components/Header/Nav/Links/index.js

```diff
@@ -1,50 +1,138 @@
 // @flow
-import styled from 'react-emotion';
+
+import { css } from 'react-emotion';
 import { connect } from 'unistore/react';
 
-import SupportTheGuardian from './SupportTheGuardian';
-import Link from './Link';
-import Search from './Search';
+import Dropdown from '@guardian/guui/components/Dropdown';
+import type { Link as DropdownLink } from '@guardian/guui/components/Dropdown';
 
-import type { LinkType } from '../__config__';
+import palette from '@guardian/pasteup/palette';
+import { textSans } from '@guardian/pasteup/fonts';
 
-const Links = styled('div')({
-    left: 0,
-    top: 0,
-    position: 'absolute',
-});
+import { tablet, desktop } from '@guardian/pasteup/breakpoints';
+
+import SupportTheGuardian from './SupportTheGuardian';
 
-const userLinks: Array<LinkType> = [
+const profileSubdomain = 'https://profile.theguardian.com';
+const subscribeUrl =
+    'https://support.theguardian.com/subscribe?INTCMP=header_support_subscribe&acquisitionData=%7B%22source%22%3A%22GUARDIAN_WEB%22%2C%22componentType%22%3A%22ACQUISITIONS_HEADER%22%2C%22componentId%22%3A%22header_support_subscribe%22%2C%22referrerPageviewId%22%3A%22jkjutjbkxfh1d8yyadfc%22%2C%22referrerUrl%22%3A%22https%3A%2F%2Fwww.theguardian.com%2Fuk%22%7D';
+const jobsUrl = 'https://jobs.theguardian.com/?INTCMP=jobs_uk_web_newheader';
+const signInUrl = `${profileSubdomain}/signin?INTCMP=DOTCOM_NEWHEADER_SIGNIN&ABCMP=ab-sign-in`;
+
+const identityLinks: Array<DropdownLink> = [
+    {
+        url: `${profileSubdomain}/user/id/123`, // TODO use actual user ID once we have a user model
```

Comment:
It might be worth checking for (adding) a route in profile that does this already, adding this link client side when we have an id will be cumbersome.

Reply:
Yeah, I've kind of deferred the actual sign-in/user credentials behaviour as wanted to think about that a bit more. I don't really know yet what a good approach is.